### PR TITLE
Add aQrator

### PR DESCRIPTION
**Pull request by WarriorGallade**

### CODE REVIEW

File: data/mods/gen9ssb/conditions.ts [hunk 1]
```diff
@@ -980,6 +992,24 @@ export const Conditions: {[k: string]: ModdedConditionData & {innateName?: strin
...
+		},
+		onStart(pokemon, source) {
+			this.add('-activate', pokemon, 'move: ' + this.effectState.sourceEffect, '[of] ' + source);
+			this.effectState.boundDivisor = source.hasItem('bindingband') ? 5 : 6;
+		},
+		onResidualOrder: 13,
+		onResidual(pokemon) {
+			this.damage(pokemon.baseMaxhp / this.effectState.boundDivisor);
+		},
+	},
```
Comment (HisuianZoroark) [inline data/mods/gen9ssb/conditions.ts:1011]: make a volatile status inside Tori's sTori instead. (see whirlpool/rise above)

File: data/mods/gen9ssb/moves.ts [hunk 2]
```diff
@@ -69,6 +69,30 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+		accuracy: 100,
+		basePower: 0,
+		category: "Status",
+		shortDesc: "Confuses the foe and deals 1/6th of its max HP for 4-5 turns.",
+		name: "Tori's Stori",
+		gen: 9,
+		pp: 5,
+		priority: 0,
+		flags: {protect: 1, reflectable: 1, mirror: 1},
+		onPrepareHit(target, source) {
```
Comment (HisuianZoroark) [inline data/mods/gen9ssb/moves.ts:83]: missing onTryMove for cancelling the tackle animation.

File: data/mods/gen9ssb/abilities.ts [hunk 3]
```diff
@@ -33,6 +33,17 @@ export const Abilities: {[k: string]: ModdedAbilityData} = {
 		gen: 9,
 	},
 
+	// aQrator
+	neverendingfhunt: {
```
Comment (HisuianZoroark) [inline data/mods/gen9ssb/abilities.ts:37]: need a flags property here (causes crashes otherwise)

**Review by HisuianZoroark — CHANGES_REQUESTED** [review]
Looks gud for a first attempt, also check the test for extra issues

File: data/mods/gen9ssb/conditions.ts (hunk 1 above)
Comment (WarriorGallade) [reply data/mods/gen9ssb/conditions.ts:1011]: good catch, i'd not noticed rise above. will do

File: data/mods/gen9ssb/moves.ts [hunk 4]
```diff
@@ -69,6 +69,50 @@ export const Moves: {[k: string]: ModdedMoveData} = {
...
+			onStart(target) {
+				this.add('-start', target, 'Neverending fHunt');
+			},
+			onResidualOrder: 6,
+			onResidual(pokemon) {
+				this.damage(pokemon.baseMaxhp / 6);
+			},
+			onEnd(target) {
+				this.add('-end', target, 'Neverending fHunt');
+			},
```
Comment (HisuianZoroark) [inline data/mods/gen9ssb/moves.ts:106]:
```suggestion
			onStart(target) {
				this.add('-start', target, 'Tori\'s Stori');
			},
			onResidualOrder: 6,
			onResidual(pokemon) {
				this.damage(pokemon.baseMaxhp / 6);
			},
			onEnd(target) {
				this.add('-end', target, 'Tori\'s Stori');
			},
```

**Review by HisuianZoroark — APPROVED** [review]

**Pull request merged**